include/endian.h

```diff
@@ -52,21 +52,50 @@
 /* Common byte swapping macros */
 
 #define __SWAP_UINT16_ISMACRO 1
-#undef  __SWAP_UINT32_ISMACRO
+#define __SWAP_UINT32_ISMACRO 1
+#define __SWAP_UINT64_ISMACRO 1
```

Comment:
let's remove all __SWAP_UINTXX_ISMACRO and libs/libc/endian/lib_swapxx.c? it doesn't make sense to keep them if we always enable the macro version.